datacontract/imports/dbt_importer.py

```diff
@@ -58,7 +59,10 @@ def import_dbt_manifest(
     """
     data_contract_specification.info.title = manifest.metadata.project_name
     data_contract_specification.info.dbt_version = manifest.metadata.dbt_version
-
+    adapter_type = manifest.metadata.adapter_type
+    data_contract_specification.servers = {
```

Comment:
This information is useful as type conversion information when exporting contracts to dbt (and dbt-sources), so we have made it output.

This information is important when exchanging multiple dbt pipelines such as data mesh with data contracts.